.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.preferences.importModuleSpecifier": "relative"
```

Comment:
This seems to be your convention, correct? VS code kept suggesting full paths when it auto-added imports, and I didn't like having to manually change them all.

Reply:
It's a limitation of typescript with nx. All nx projects share the same typescript paths because that's how you reference other projects, this means you can't have aliases scoped to each project. So that means relative imports for internal dependencies and absolute for external dependencies. Perhaps this should be `non-relative` or `project-relative`.

The other question is whether we want to save vscode settings in the repo at all? I'm not sure what the right call is here. Open to your thoughts.

Reply:
It's just a simple matter of discernment. For vs code settings that will help us write consistent code across developers, I don't see why there would be any problem. We're not going to add every vs code setting here.